ci/obd-demo.yaml

```diff
@@ -328,3 +311,24 @@ spec:
       targetPort: 8070
       protocol: TCP
       appProtocol: HTTP
+
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    kardinal.dev.service/ingress: "true"
+  name: ingress
+spec:
+  rules:
+    - host: prod.app.localhost
```

Comment:
shouldn't it be `baseline.app.localhost` now that the default flow name changed?